### ISSUE TITLE
how to reuse active voice matching?

### ISSUE THREAD

**Issue by etanter** [otto-torino/django-baton#154]
Hi @abidibo,

Given a URL and a menu (baton conf), I'd like to check whether there will be one active voice for that URL.
(as a way of checking that the URL that is requested is 'exposed' by the menu)

You are surely doing this process already in order to build the representation of the menu with the properly expanded menu and highlighted voice.

Is this check logic exposed somehow? Otherwise, could you make it accessible? 

Basically, I'd like to be able to do:
```
url = ...requested url...
menu = ... get_menu(request)...
if is_valid(url, menu):
  ... go to url ...
else:
  ... redirect to somewhere else...
```

Thanks!

**Comment by etanter** [comment]
closing this since I found another way to achieve what I was after. Sorry for the noise!
